tdrs-backend/tdpservice/email/helpers/data_file.py

```diff
@@ -0,0 +1,20 @@
+"""Helper functions for sending data file submission emails."""
```

Comment:
looks like this test was left behind.

```py
@pytest.mark.django_db
def test_send_data_submitted_email(mocker):
    """Test that the send_data_submitted_email function runs."""
    mocker.patch(
        'tdpservice.email.email_helper.automated_email.delay',
        return_value=None
    )
    context = {
        'stt_name': 'foo_stt',
        'submission_date': 'foo_date',
        'submitted_by': 'foo_user',
    }
    send_data_submitted_email(context)

    assert tdpservice.email.email.automated_email.delay.called_once_with(
        email_context=context,
    )
    ```
    
    Although I am not sure how useful it is?

Reply:
yeah i took it out, it doesn't really test anything since we moved the logic over to the views/models